maggy/core/environment/hopsenvironment.py

```diff
@@ -0,0 +1,242 @@
+import hsfs
+import json
+import os
+
+from hops import constants
+from hops import hdfs as hopshdfs
+from hops import util as hopsutil
+from hops.experiment_impl.util import experiment_utils
+from maggy import tensorboard
+from maggy import util
+from maggy.core.environment.abstractenvironment import AbstractEnvironment
+
+
+class HopsEnvironment(AbstractEnvironment):
```

Comment:
Please don't call it `Hops` because with Hops we usually refer to the FileSystem HopsFS, that can lead to confusion because it doesn't directly have to do with the FS. Instead call it `Hopsworks`